solutions/usaco-987.mdx

```diff
@@ -53,4 +53,44 @@ int main(){
 
 </CPPSection>
 
+<PySection>
+
+## Implementation
+
+```python
+import sys
+
+sys.stdin = open("word.in", "r")
+sys.stdout = open("word.out", "w")
+input = sys.stdin.readline
+
+def bessie_word(n, k, t):
+
+	# split string into list
+	n_list = t.split()
+
+	result = [n_list[0]]
+	cur_idx = 0
+	word_cnt = len(result[cur_idx])
```

Comment:
(optional) please use something other than `cnt`

Reply:
but is it not word count

Reply:
idk just tell me what to change it to pls